release-team/README.md

```diff
@@ -188,6 +195,69 @@ For all unplanned or embargoed releases
 
 ---
 
+## Release Team Selection
+
+In addition to discharging the duties of their respective Release Team roles, the current Release Team is charged with selecting the team for the following release cycle.
+
+Team selection should happen transparently.
+
+In the 1.11 release cycle, we selected the Release Team Lead by quorum during a public Release Team meeting and additional roles were staffed via [an issue in k/sig-release](https://github.com/kubernetes/sig-release/issues/167) and a set of PRs, starting with [this one](https://github.com/kubernetes/sig-release/pull/168).
+
+Each Release Team role is responsible for staffing their respective role, with this order of fall-through in mind:
+- training and selecting a successor from the current pool of role shadows
+- training and selecting a successor from non-Release Team members
+- staffing the role themselves
+
+Ultimately, if none of these can be satisfied, responsibility falls to the future Release Team Lead and SIG Release to staff the roles.
+
+### Release Team Lead Selection
+
+The incoming Release Team Lead _MUST_ have participated on the Release Team for 2 or more release cycles, acting in a lead (non-shadow) capacity for at least one of those cycles.
+
+Release Team Leads should be staffed, with this order of fall-through in mind:
+- the current pool of Release Team Lead shadows
+- the current pool of Release Team members
+- former Release Team members
+
+Bear in mind that these are suggestions based on precedent and a Release Team Lead may be nominated by any Release Team member, past or present.
+
+_The new Release Team Lead should be agreed by a majority of the current Release Team members._
+
+**If for some reason a new RT Lead cannot be selected by these methods, the current Release Team Lead should be prepared to continue to serve in the following release cycle until a successor can be found.**
```

Comment:
I would strike this given the above. We did actually try to have this as a requirement but it was deemed unfair to the release team lead who shouldn't really be held responsible for a failure of the team selection process